app.ytmdesktop.ytmdesktop.metainfo.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>app.ytmdesktop.ytmdesktop</id>
+  
+  <name>ytmdesktop</name>
+  <summary>Free cross platform Desktop Player for YouTube Music</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>CC0-1.0</project_license>
+
+  <content_rating type="oars-1.1" />
+
+  <recommends>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </recommends>
+  
+  <description>
+    <p>
+      YTMDesktop have NO affiliation with Google or YouTube. YouTube Music are trademark of Google Inc.
```

Comment:
Whilst this is good to include you should also include a description that actually tells users what this app is and what it does otherwise they won't be that keen to install it.